lib/src/base-styles/_nx-tables.scss

```diff
@@ -119,10 +119,16 @@
   position: relative;
   z-index: 0;
 
-  .nx-cell--header {
+  > thead tr:nth-child(1) th {
     position: sticky;
     top: 0;
-    z-index: 1;
+    z-index: 1
```

Comment:
Missing semi-colon here and on the other one below

Reply:
not longer needed with: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/a954210b17845f9ed60e40fd1bf15de1f5723c9a